spec/admin/emotion_index_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
```

Comment:
The location of these tests should match where they are relative to the `app/` folder, in other words `spec/reports/emotion_index_spec.rb` rather than the admin folder.